src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -715,6 +715,10 @@ The default is 'false.' However, when targeting .NET 7 or lower, the default is
   <data name="ResponseFileNotFound" xml:space="preserve">
     <value>Response file '{0}' does not exist.</value>
   </data>
+  <data name="ShortWorkloadSearchVersionDescription" xml:space="preserve">
+    <value>Search for available workload versions or what comprises a workload version. Use 'dotnet workload search version --help' for more information.</value>
+    <comment>dotnet workload search version --help should not be localized.</comment>
```

Comment:
nit: use the [locking syntax](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/Localization.md#locking-translations) for pinned translation strings so that the Loc team's tooling helps prevent unintentional translation.